Scemantic Focus

```diff
@@ -0,0 +1,110 @@
+---
+id: 
+name: Scemantic Focus 
+rule_type: atomic
+
+description: | 
+ This rule checks that every element part of the sequential focus navigation has a semantic role.
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+
+authors:
+- Carlos Duarte
+- Damien Engels
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+The rule applies to all element in the sequential focus navigation excepts scrollable content element.
+
+## Expectation
+
+Target element has a semantic role.
+
+## Assumptions
+
+- Giving focus to scrollable content elements is the only way to scroll non interactive content using a keyboard.
```

Comment:
I don't think I understand why this is an assumption. Can you clarify?

Reply:
It would have been better to have this as note in the applicability instead of an assumption. But after reviewing the definition of focusable are I don't think this is needed, given that if a scrollable region of an element is focusable, then it should have a semantic role so that users of AT can understand what role that region plays.